lib/grinder_files.dart

```diff
@@ -98,12 +100,210 @@ class FileSet {
 }
 
 /**
+ * A class to make it easier to manipulate file system entites. Once paths or
+ * entites are converted into `Path`s, they can be easily copied, deleted,
+ * joined, and their name retrieved.
```

Comment:
Or keep the functionality, but implement it with the `path` package to decrease likelihood of bugs.